src/Microsoft.ML.OnnxTransform/doc.xml

```diff
@@ -0,0 +1,92 @@
+          The TensorflowTransform has following assumptions regarding the input, output and processing of data.
+        </para>
+        <list type="number">
+          <item>
+            <description>
+              The transform currently accepts the <a href="https://www.tensorflow.org/mobile/prepare_models">frozen TensorFlow model</a> file as input.
+            </description>
+          </item>
+          <item>
+            <description>The transform supports scoring only one example at a time.</description>
+          </item>
+          <item>
+            <description>The name of input column(s) should match the name of input(s) in Tensorflow model.</description>
+          </item>
+          <item>
+            <description>The name of each output column should match one of the operations in the Tensorflow graph.</description>
+          </item>
+          <item>
+            <description>Currently, float and double are the only acceptable data types for input/output.</description>
+          </item>
+          <item>
+            Upon success, the transform will introduce a new column in <see cref="IDataView"/> corresponding to each output column specified.
+          </item>
+        </list>
+
+        The inputs and outputs of a TensorFlow model can be obtained using the <a href="https://github.com/tensorflow/tensorflow/tree/master/tensorflow/tools/graph_transforms/README.md#inspecting-graphs">
+          <code>summarize_graph</code> tool
+        </a>.
+
+      </remarks>
+      <example>
+        <code language="csharp">
+          pipeline.Add(new TextLoader(dataFile).CreateFrom&lt;MNISTData&gt;(useHeader: false));
+          pipeline.Add(new ColumnCopier(("NumericImageVec", "Input");
+          pipeline.Add(new TensorFlowScorer()
+          {
+            ModelFile = model_location;
+            InputColumns = new []{ "Input" };
+            OutputColumns = "Output"
+          }
+        </code>
+      </example>
+      <example>
+        <code language="csharp">
+          var pipeline = new LearningPipeline(seed: 1);
+          pipeline.Add(new TextLoader(dataFile).CreateFrom&lt;CifarData&gt;(useHeader: false));
+          pipeline.Add(new ImageLoader(("ImagePath", "ImageReal"))
+          {
+            ImageFolder = imageFolder
+          });
+
+          pipeline.Add(new ImageResizer(("ImageReal", "ImageCropped"))
+          {
+            ImageHeight = imageHeight,
+            ImageWidth = imageWidth,
+            Resizing = ImageResizerTransformResizingKind.IsoCrop
+          });
+
+          pipeline.Add(new ImagePixelExtractor(("ImageCropped", "Input"))
+          {
+            UseAlpha = false,
+            InterleaveArgb = true
+          });
+
+          pipeline.Add(new TensorFlowScorer()
+          {
+            ModelFile = model_location,
+            InputColumns = new[] { "Input" },
+            OutputColumns = "Output"
+          });
```

Comment:
same

Reply:
replaced code with unit test snippet to ensure compilation

---
In reply to: [220397137](https://github.com/dotnet/machinelearning/pull/942#discussion_r220397137) [](ancestors = 220397137)